cyipopt/scipy_interface.py

```diff
@@ -522,7 +522,9 @@ def minimize_ipopt(fun,
     # Rename some default scipy options
     replace_option(options, b'disp', b'print_level')
     replace_option(options, b'maxiter', b'max_iter')
-    if b'print_level' not in options:
+    if getattr(options, 'print_level', False) is True:
```

Comment:
When `disp` is `True`, we get an error because `print_level` needs to be a bool.